server.go

```diff
@@ -25,6 +25,7 @@ type Server struct {
 	MaxIdleSeconds    int
 	MaxMessageBytes   int
 	AllowInsecureAuth bool
+	StrictSMTP        bool
```

Comment:
Since this is the `smtp` package, is it possible to rename this to simply `Strict`?

Reply:
Renamed.